packages/flutter/lib/src/material/about.dart

```diff
@@ -211,6 +213,61 @@ void showAboutDialog({
   );
 }
 
+/// Displays an [AboutDialog], which describes the application and provides a
+/// button to show licenses for software used by the application.
```

Comment:
The first sentence of the doc comments should mention that it adapts based on the platform.